File: specification/specs/x_pack/machine_learning/delete_forecast/DeleteForecastRequest.ts
@rest_spec_name("ml.delete_forecast")
class DeleteForecastRequest extends RequestBase {
  path_parts?: {
    job_id: Id;
    forecast_id?: Id;
  }
  query_parameters?: {
    allow_no_forecasts?: boolean;
    timeout?: Time;
  }
  body?: {
  }
}

File: output/csharp/XPack/MachineLearning/DeleteForecast/DeleteForecastRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class DeleteForecastRequest : RequestBase {
		
		[DataMember(Name="allow_no_forecasts")]
		public bool AllowNoForecasts { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }

	}
}
